.github/integration/tests/sda/10_upload_test.sh

```diff
@@ -61,38 +61,4 @@ if [ "$num_log_rows" -ne 12 ]; then
     exit 1
 fi
 
-## test with token from OIDC service
-echo "testing with OIDC token"
```

Comment:
what's the reason for deleting these tests?

Reply:
The mock oidc service (the pyhon one) is the one that is replaced by the mock-aai.
And since we can't programatically extract the second token they aren't needed.

It is possible to perform the same test (upload with a different user) but it has to be done manually.